app/controllers/contributions_controller.rb

```diff
@@ -15,6 +15,26 @@ def index
     end
   end
 
+  def show
+    contribution = Listing.find(params[:id])
+
+    render(
+      :show,
+      locals: {
+        contribution: contribution,
+      }
+    )
+  end
+
+  def claim_ask_form
+    contribution_id = params[:id]
+    render locals: { contribution_id: contribution_id }
+  end
+
+  def claim_ask
```

Comment:
Should requests to this have an accompanying example?

Reply:
Moved this method to `claims_controller` which is a nested resource of contributions and it is RESTful now. do we still need an example?